sw/device/tests/BUILD

```diff
@@ -181,7 +181,11 @@ opentitan_test(
 opentitan_test(
     name = "alert_handler_lpg_clkoff_test",
     srcs = ["alert_handler_lpg_clkoff_test.c"],
-    cw310 = cw310_params(timeout = "moderate"),
+    cw310 = cw310_params(
+        timeout = "moderate",
+        # This test is slow; run it only on the nightly CI.
```

Comment:
Nit: I'm not sure that "nightly CI" makes sense. Maybe "nightly regression" or something work work better?